src/main/java/com/cegeka/academy/web/rest/GlobalExceptionHandler.java

```diff
@@ -16,4 +20,16 @@
         return new ResponseEntity<String>(exception.getMessage(), HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    public ResponseEntity<String> handleValidationExceptions(MethodArgumentNotValidException ex) {
+        List errors = new ArrayList();
+        ex.getBindingResult().getAllErrors().forEach((error) -> {
+            String errorMessage = error.getDefaultMessage();
+            errors.add(errorMessage);
+        });
+
+        return new ResponseEntity(errors.toString(), HttpStatus.NOT_FOUND);
+
```

Comment:
Statusul NOT_FOUND este pentru lipsa de resurse. In cazul tau, arunci exceptia cand o validare nu este ok, ceea ce inseamna ca a gresit cel care te apeleaza. In cazul asta ar trebui sa arunci BAD_REQUEST